DS-Session-1-Assignment/src/question3/Term.java

```diff
@@ -0,0 +1,26 @@
+package question3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class will represent a single term in multivariable polynomial with integer as its coefficient and Variable type list as its variables 
+ * @author Jugal
+ * Dated 07/30/2019
+ */
+public class Term {
```

Comment:
Use a class, name as Polynomial and it will contain the Variable list. And you will form the Polynomial using this class.